utils/schemas/main_config.json

```diff
@@ -347,29 +299,86 @@
       "title": "The icon schema",
       "description": "An explanation about the purpose of this instance.",
       "default": "",
-      "examples": [
-        "icon.jpg"
-      ]
+      "examples": ["icon.jpg"]
     },
     "logo": {
       "$id": "#/properties/logo",
       "type": "string",
       "title": "The logo schema",
       "description": "An explanation about the purpose of this instance.",
       "default": "",
-      "examples": [
-        "logo.png"
-      ]
+      "examples": ["logo.png"]
     },
     "website": {
       "$id": "#/properties/website",
       "type": "string",
       "title": "The website schema",
       "description": "An explanation about the purpose of this instance.",
       "default": "",
+      "examples": ["https://www.newrelic.com"]
+    },
+    "documentation": {
+      "$id": "#/properties/documentation",
+      "type": "array",
+      "title": "The documentation schema",
+      "description": "List of documentation links for this pack.",
+      "default": [],
+      "uniqueItems": true,
       "examples": [
-        "https://www.newrelic.com"
-      ]
+        [
+          {
+            "name": "How to use this pack",
+            "url": "https://example.com",
+            "description": "A brief summary of what this doc entails."
+          }
+        ]
+      ],
+      "items": {
+        "$id": "#/properties/documentation/items",
+        "anyOf": [
+          {
+            "$id": "#/properties/documentation/items/anyOf/0",
+            "type": "object",
+            "title": "The first anyOf schema",
```

Comment:
I think this might be left over from copy paste!

Reply:
which part? the `id` is unique. and i just took the generic `title` value like the others

Reply:
Gotcha! could you just update the `description`? Maybe something like "a collection of name, url, and optional description for a piece of documentation"

Reply:
added that just now